### ISSUE TITLE
bug: sticking

### ISSUE THREAD

**Issue by Nalmor1** [anddea/revanced-patches#481]
### Type

Error at runtime

### Tools used

RVX Manager

### Application

YouTube 19.16.39. Patches-v.2.225.0 dev.14

https://github.com/anddea/revanced-patches/assets/169981296/662885cb-1ae5-44d3-a3de-30e0f5225c35

  by anddea

### Bug description

Hello, this problem appeared on versions 19.16.38-39, on version 19.11.43 there was no such problem, the application seems to be stuck and does not allow me to scroll the tape, while this happened to me once, but maybe it will happen again.

### Device Environment

Asus ZenFone 9

### Acknowledgements

- [X] This issue does not reproduce on unpatched YouTube or YT Music.
- [X] This issue is not a duplicate of an existing bug report.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
- [X] I have written the title and contents in English.

**Comment by anddea** [comment]
Can you still reproduce this issue? Any steps to reproduce? Can you reproduce it with inotia's patches?

**Comment by Nalmor1** [comment]
I tried applying your patches again and the problem remained but only once, then I restarted the app and the problem disappeared.
The video switch buttons are also stuck, I had this before but it didn't bother me, it only appears the first time I start the app, restarting also helps.also it's on annotia00 patches
![Screenshot_20240519-225538_YouTube](https://github.com/anddea/revanced-patches/assets/169981296/3550dba0-7ad0-47f6-9116-51e87dafe51f)

**Comment by Nalmor1** [comment]
![Screenshot_20240519-225538_YouTube_1](https://github.com/anddea/revanced-patches/assets/169981296/c849eba6-e09e-4e3b-a7e0-f664cdb51dab)

**Comment by anddea** [comment]
> not shared with you?

No, I think they misunderstood you. All changes from intoia's repo is already in this repo as well, unless I missed something, that's why I asked you could you reproduce with intoia's patches. To be honest I never noticed such an issue, that might be OS related somehow but if it happens only for the first launch it's even more weird.

**Comment by Nalmor1** [comment]
I’ll keep an eye on it, if I write anything here, thanks for your work, and by the way, I tried your latest patch on version 19.17.41, it works without problems, or can you recommend a better version?

**Comment by anddea** [comment]
If it works for you it's fine to use, I just didn't test it on newer versions myself yet.
